### ISSUE TITLE
Conflict with Semantic UI

### ISSUE THREAD

**Issue by daxpandhi** [francoischalifour/medium-zoom#165]
When using medium-zoom with Semantic UI, the code works but images disappear when zooming.

This is because of a conflict with the z-index set by Semantic UI's "pusher" element for off-screen navigation which uses a Z-index value of 2. (https://semantic-ui.com/modules/sidebar.html#/usage)

I tried this fix:

```
.medium-zoom-overlay, .medium-zoom-image--opened {
    z-index: 999;
}
```

Might it be worth adding to the injected css in the script itself, perhaps?

**Comment by francoischalifour** [comment]
The default `z-index` value is quite low to not be too intrusive. Since that's something that each person can update easily to their own website's needs, we won't increase it for now.

**Comment by daxpandhi** [comment]
Makes sense. But since this is by design, yet affecting an entire framework and not just a custom site, and was difficult to figure out, perhaps it might be worth mentioning in the documentation somewhere and the specific classes that may need a higher z-index. I only found it after a fair bit of trial and error. On the surface, it just makes it look like the script is broken.

**Comment by francoischalifour** [comment]
That's fair, thanks! Addressed in #167.
